modules/ols-configuring-lightspeed-with-a-trust-provider-certificate-for-the-llm-using-console.adoc

```diff
@@ -0,0 +1,73 @@
+// This module is used in the following assemblies:
+
+// * configure/ols-configuring-openshift-lightspeed.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="ols-configuring-openshift-lightspeed-with-a-trust-certificate-required-by-llm-provider-using-console_{context}"]
+= Configuring {ols-long} with a trust provider certificate for the LLM using the web console
+
+This procedure explains how to configure {ols-long} with a trust provider certificate for the Large Language Model (LLM) provider using the .
+
+[NOTE]
+====
+If the LLM provider you are using requires a trust certificate to authenticate the {ols-long} service you must perform this procedure. If the LLM provider does not require a trust certificate to authenticate the service then you should skip this procedure.
+====
+
+.Prerequisties
+
+* You are logged in to the {ocp-product-title} web console as `cluster-admin`.
+
+* The {ols-long} Operator is installed.
+
+.Procedure
+
+. Click *Add* in the upper-right corner of the {ocp-product-title} web console.
+
+. Copy the contents of the certificate file and paste it into a file called `caCertFileName`.
+
+. Click *Create*.
+
+. Click *Add* in the upper-right corner of the {ocp-product-title} web console.
+
+. Create a `ConfigMap` object called `trusted-certs`. 
++
+.Example file
+[source,terminal]
+----
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  name: trusted-certs
+  namespace: openshift-lightspeed
+data:
+  caCertFileName: |
+    -----BEGIN CERTIFICATE-----
+    .
+    .
+    .
+    -----END CERTIFICATE-----  
+----
+
+. Update the `OLSConfig` custom resource file to include the name of the `ConfigMap` object that contains one or more CA certificates required to connect to your LLM provider.
++
+.Example {rhelai} CR file
+[source,yaml,subs="attributes,verbatim"]
+----
+apiVersion: ols.openshift.io/v1alpha1
+kind: OLSConfig
+metadata:
+  name: cluster
+spec:
+  ols:
+    defaultProvider: rhelai
+    defaultModel: models/granite-7b-redhat-lab
+    additionalCAConfigMapRef:
+      name: trusted-certs
```

Comment:
Do we need a call-out here or are folks smart enough to know where to put the config map name? (Assuming I am, that is.)

Reply:
done